ambry-network/src/test/java/com.github.ambry.network/NetworkClientTest.java

```
@@ -196,7 +196,7 @@ public void testExceptionOnConnect() {
     selector.setState(MockSelectorState.ThrowExceptionOnConnect);
     try {
       networkClient.sendAndPoll(requestInfoList, 100);
-    } catch (IOException e) {
+    } catch (Exception e) {
```

why catch and assert? Just let the exception fail the test?